services/core-web/src/components/mine/Permit/MinePermitTable.js

```diff
@@ -179,26 +179,55 @@ const columns = [
           </div>
         </Menu>
       );
-      return (
-        <AuthorizationWrapper
-          permission={Permission.EDIT_PERMITS}
-          isMajorMine={text.major_mine_ind}
+
+      const deletePermitPopUp = (
+        <Popconfirm
+          placement="topLeft"
+          {...(() => {
+            return record.permit.bonds && record.permit.bonds.length > 0
+              ? {
+                  title: "You can not delete permit with bonds",
+                  okText: "Ok",
```

Comment:
"You cannot delete a permit that has associated bond records"

Reply:
done